### PR TITLE
W-15577052 - RPA Manager release notes 2024-04/2.

### PR DESCRIPTION
**Pull request by jhammann72**

### CODE REVIEW

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 1]
```diff
@@ -2,6 +2,34 @@
...
+[[april-22-2024]]
+== April 22, 2024
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| A long Japanese process name will now be displayed properly on a process tile. | W-14835485
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| Process names with Japanese characters no longer get split on a process tile. | W-14835485
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 2]
```diff
@@ -2,6 +2,34 @@
...
+== April 22, 2024
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| Process names with Japanese characters no longer get split on a process tile. | W-14835485
+| The content of table cells will now be displayed properly with respect to the respective column width. | W-15242583
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| Content of table cells no longer gets truncated when there's more space available. | W-15242583
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 3]
```diff
@@ -2,6 +2,34 @@
...
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| Process names with Japanese characters no longer get split on a process tile. | W-14835485
+| Content of table cells no longer gets truncated when there's more space available. | W-15242583
+| The text of downloadable applications on the RPA Manager's homepage will be shown correctly formatted. | W-15312305
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| The text of downloadable applications on the RPA Manager's homepage no longer extends outside the buttons. | W-15312305
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 4]
```diff
@@ -2,6 +2,34 @@
...
+== April 22, 2024
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| Process names with Japanese characters no longer get split on a process tile. | W-14835485
+| Content of table cells no longer gets truncated when there's more space available. | W-15242583
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| The content of table cells no longer gets truncated when there's more space available. | W-15242583
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 5]
```diff
@@ -2,6 +2,34 @@
...
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| Process names with Japanese characters no longer get split on a process tile. | W-14835485
+| The content of table cells no longer gets truncated when there's more space available. | W-15242583
+| The text of downloadable applications on the RPA Manager's homepage no longer extends outside the buttons. | W-15312305
+| Now the field *To (email)* in an _Alert Target_ will be enforced as a mandatory field. | W-11716347
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| The field *To (email)* in an Alert Target is now enforced as a mandatory field. | W-11716347
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 6]
```diff
@@ -2,6 +2,34 @@
...
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| Process names with Japanese characters no longer get split on a process tile. | W-14835485
+| The content of table cells no longer gets truncated when there's more space available. | W-15242583
+| The text of downloadable applications on the RPA Manager's homepage no longer extends outside the buttons. | W-15312305
+| The field *To (email)* in an Alert Target is now enforced as a mandatory field. | W-11716347
+| The email address in an _Alert Target_ will be verified. | W-11720892
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| The email address in an Alert Target is now verified. | W-11720892
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 7]
```diff
@@ -2,6 +2,34 @@
...
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| Process names with Japanese characters no longer get split on a process tile. | W-14835485
+| The content of table cells no longer gets truncated when there's more space available. | W-15242583
+| The text of downloadable applications on the RPA Manager's homepage no longer extends outside the buttons. | W-15312305
+| The field *To (email)* in an Alert Target is now enforced as a mandatory field. | W-11716347
+| The email address in an Alert Target is now verified. | W-11720892
+| Modifying an inactive _Alert Rule_ will now keep the rule inactive. | W-15424278
```
Comment (Cristian-Venticinque) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| Modifying an inactive Alert Rule no longer activates it. | W-15424278
```

File: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc [hunk 8]
```diff
@@ -2,6 +2,34 @@
...
+
+// == Fixed Issues
+include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]
+
+[%header,cols="6a,1a"]
+|===
+|Issue Resolution |ID
+| Process names with Japanese characters no longer get split on a process tile. | W-14835485
+| The content of table cells no longer gets truncated when there's more space available. | W-15242583
+| The text of downloadable applications on the RPA Manager's homepage no longer extends outside the buttons. | W-15312305
```
Comment (Crispy-Salesforce) [inline rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]:
```suggestion
| The text of downloadable applications on the RPA Manager homepage no longer extends outside the buttons. | W-15312305
```

Comment (Crispy-Salesforce) [reply rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc]: As per our [Style Guide](https://confluence.internal.salesforce.com/pages/viewpage.action?spaceKey=MTDT&title=MuleSoft+CX+Writing+Style+Reference), at "possessives (‘s)"

**Review by Crispy-Salesforce — DISMISSED** [review]

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**